### PR TITLE
Add Cybersecurity Blue Team

### PR DESCRIPTION
**Pull request by meitar**
https://github.com/meitar/awesome-cybersecurity-blueteam#readme

This list is focused on curating a big picture overview of tools and
resources dealing with collaborative security flaw identification,
remediation, and verification. It is a relatively new list that provides
a holistic subset of several other security-related lists along with a
number of new items focused primarily around the needs of DevSecOps/Blue
Team needs in an organization. Relevant other lists, such as awesome IR,
awesome pentest, etc., are linked where relevant so as not to duplicate
work or reinvent the wheel. (Instead, I contribute to those other
lists directly when more appropriate than adding to this list.)

Thanks for the consideration.

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by hsiaoyi0504** [comment]
It looks like one of item of the list doesn't exist anymore: https://nccgroup.github.io/Scout2/.

Can you check again ?

**Comment by meitar** [comment]
Weird, it must have been recently removed. It's still in [Google's cache](https://nccgroup.github.io/Scout2/) and the [Web Archive](https://web.archive.org/web/20180429203531/https://nccgroup.github.io/Scout2/) is also aware of it. `¯\_(ツ)_/¯`

I'll update the link in the awesome list. Thanks.

**Comment by meitar** [comment]
@MFatihMAR Looks like many if not all of these linting issues are new requirements which differ from the guidelines in this repository's pull request template.

I'm going to wait with these until y'all figure out how you actually want people to contribute, since I've been getting conflicting information on numerous of my PRs and I don't care quite so much as to continue to go back-and-forth between sometimes contradicting requirements.

Good luck getting the dust to settle. :)

**Comment by sindresorhus** [comment]
@meitar The output is better now:

```
❯ awesome-lint https://github.com/meitar/awesome-cybersecurity-blueteam
✖ Linting

  README.md:16:5
  ✖    1:36  Invalid badge source                                                                                remark-lint:awesome/badge
  ✖   16:5   Incorrect indentation before bullet: remove 2 spaces                                                remark-lint:list-item-bullet-indent
  ✖   17:5   Incorrect indentation before bullet: remove 2 spaces                                                remark-lint:list-item-bullet-indent
  ✖   18:5   Incorrect indentation before bullet: remove 2 spaces                                                remark-lint:list-item-bullet-indent
  ✖   21:5   Incorrect indentation before bullet: remove 2 spaces                                                remark-lint:list-item-bullet-indent
  ✖   28:1   ToC item "Transport-layer defense" does not match corresponding heading "Transport-layer defenses"  remark-lint:awesome/toc
  ✖   80:4   Text "OSX" should be written as "macOS"                                                             remark-lint:awesome/spell-check
  ✖   81:4   Text "OSX" should be written as "macOS"                                                             remark-lint:awesome/spell-check
  ✖  159:81  List item description must end with proper punctuation                                              remark-lint:awesome/list-item

  9 errors
```

https://github.com/sindresorhus/awesome-lint

File: readme.md [hunk 1]
```diff
@@ -433,6 +433,7 @@
 - [Vehicle Security and Car Hacking](https://github.com/jaredthecoder/awesome-vehicle-security#readme)
 - [Web Security](https://github.com/qazbnm456/awesome-web-security#readme) - Security of web apps & services.
 - [Lockpicking](https://github.com/meitar/awesome-lockpicking#readme) - The art of unlocking a lock by manipulating its components without the key.
+- [Blue Teams](https://github.com/meitar/awesome-cybersecurity-blueteam#readme) - Tools for collaborative IT system security flaw detection, remediation, and verification.
```
Comment (sindresorhus) [inline readme.md]: The title here should match the list title: `Cybersecurity Blue Team`

Comment (sindresorhus) [reply readme.md]: I think the description could be improved. It should not mention `Tools`.

How about this?

> Groups of individuals who identify security flaws in information technology systems.

**Comment by meitar** [comment]
@sindresorhus That description looks good to me. I made a few additional changes to fix several linting errors. However, I don't understand what "Invalid badge source" means and I also don't want to change the `OSX` to `macOS` text because these reference tools whose names are OSX and cannot be changed on my list until the upstream project changes their name (which is unlikely). It would be an error for me change their name so I hope you'll ignore these specific linting errors in this case.

**Comment by meitar** [comment]
Oh, I figured out what is meant by invalid badge source, and fixed it, I think.

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1115187993239625728

**Pull request merged**